client/src/components/game/Game.tsx

```diff
@@ -164,6 +163,11 @@ function Game() {
       <Dialog isOpen={isHelpOpen} onClose={closeHelp}>
         <Help />
       </Dialog>
+      {isInLobby && (
+        <Dialog isOpen={isInviteDialogOpen} onClose={closeInviteDialog}>
+          <InviteDialogContent />
+        </Dialog>
```

Comment:
Do you think we should move this to a component called `InviteDialog` instead of `InviteDialogContent`?

Reply:
Yeah, that is a great idea